### DISCUSSION TITLE
How do EXIF and filename/folder tiimestamps relate in GoProx? Should they all match?

### DISCUSSION THREAD

**Discussion by victorhooi** [fxstein/GoProX#58] [General]
Hi,

I'm just trying to understand how timestamps are intended to work with GoProX - in terms of:

* EXIF data timestamp
* Modification and creation time on the files
* Filename timestamp in the copied files
* Folder datestamp in the copied files

Is the idea that GoProX is meant to take the EXIF data timestamp , and then apply this to the filenames and folder names that it creates?

Because I seem to be noticing some incongruity on my side - but not sure if that's intended behaviour or not?

**Comment by fxstein** [comment]
All file system dates should be coming from the EXIF data.

If you copy folders around the filesystem dates easily get destroyed.

Hope this helps!
